src/AdminRoom.ts

```diff
@@ -126,16 +130,30 @@ export class AdminRoom extends AdminRoomCommandHandler {
         });
     }
 
-    @botCommand("help", "This help text")
+    @botCommand("help", { help: "This help text", category: "general" })
     public async helpCommand() {
         const enabledCategories = [
+            "general",
             this.config.github ? "github" : "",
             this.config.gitlab ? "gitlab" : "",
             this.config.jira ? "jira" : "",
         ];
         return this.botIntent.sendEvent(this.roomId, AdminRoom.helpMessage(undefined, enabledCategories));
     }
 
+    @botCommand("disconnect", { help: "Remove a connection", requiredArgs: ['roomId', 'id'], category: "general" })
+    public async disconnect(roomId: string, id: string) {
+        // it's stupid that we need roomId -- shouldn't `id` identify the connection?
```

Comment:
ID is derived from `roomId` *and* it should be unique, so aside of theoretical md5 conflicts we should just be able to change this API, I think.

Reply:
I'd be up for that change, I'd imagine conflicts are fairly rare here. We should explode in fire if there *IS* a conflict though so we don't end up deleting the wrong one (and they can file a bug report with us, or delete it manually).